### PR TITLE
Updating vars in various components and adding analytics payload data.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR updates some stragglers related to the analytics events changes!

### Any background context you want to provide?

🌵 

### What are the relevant tickets/cards?

Refs [Pivotal ID #166595074](https://www.pivotaltracker.com/story/show/166595074)

### CODE REVIEW

File: resources/assets/components/Quiz/Quiz.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ class Quiz extends React.Component {
     }
 
     trackAnalyticsEvent({
-      context: { responses: this.state.choices },
```
Comment (mendelB) [inline resources/assets/components/Quiz/Quiz.js]: Just confirming that we don't have any live quizzes relying on this Puck data?

Comment (weerd) [reply resources/assets/components/Quiz/Quiz.js]: 🤷‍♂

Comment (weerd) [reply resources/assets/components/Quiz/Quiz.js]: Data seemed to want to remove, but I'll confirm now.

Comment (weerd) [reply resources/assets/components/Quiz/Quiz.js]: This turned out to be a hairy question... 😬

Comment (mendelB) [reply resources/assets/components/Quiz/Quiz.js]: I just love complicating things 😈

Comment (weerd) [reply resources/assets/components/Quiz/Quiz.js]: confirmed! we're removing them 🎉

**Pull request merged**